### DISCUSSION TITLE
Documentation for workerd

### DISCUSSION THREAD

**Discussion by KianNH** [cloudflare/workerd#666] [General]
This discussion is purely just to see what direction the Workers team was taking with documentation for `workerd` specifically. A few ideas off the top of my head:

* A more in-depth guide for configs to build on-top of the example in the [README](https://github.com/cloudflare/workerd/blob/main/README.md).
* Using `workerd compile` or creating the configs using a [Cap'n Proto serialisation library](https://capnproto.org/otherlang.html) in a given language. 
* Actually running `workerd`, expanding on the `systemd` example.
* Building `workerd`, where to get pre-built binaries from (`npm`), etc.

Mainly, if that sort of content is wanted and if it'd go in the Cloudflare developer documentation or just in the `/docs/` folder in this repository.

Lots of the information already exists in various documentation around the repo but expanding on those with more advanced examples/use-cases would be awesome.

**Comment by kentonv** [comment]
Agreed we need to do better here.

I don't have strong opinions on how to do it, though.

Maybe @jasnell and/or @mrbbot have thoughts.
